reV/handlers/multi_year.py

```diff
@@ -417,13 +473,18 @@ def collect_profiles(cls, my_file, source_files, dset, group=None):
         ----------
         my_file : str
             Path to multi-year .h5 file
-        source_files : list
-            List of .h5 files to collect datasets from
+        source_files : list | str
+            List of .h5 files to collect datasets from. This can also be a
+            unix-style /filepath/pattern*.h5 to find .h5 files to collect,
+            however all resulting files must be .h5 otherwise an exception will
+            be raised. NOTE: .h5 file names much indicate the year the data
```

Comment:
```suggestion
            be raised. NOTE: .h5 file names must indicate the year the data
```

Man, it really is a pain to have to maintain the same docstring in so many places....

Reply:
Eh its not so bad if you just do some find/replace magic. Stupid typos are always annoying though haha

Reply:
Fair enough